lowcar/devices/KoalaBear/KoalaBear.h

```diff
@@ -1,35 +1,40 @@
 #ifndef KOALABEAR_H
 #define KOALABEAR_H
 
+#include <SPI.h>
```

Comment:
why is this needed? I don't see any SPI devices

Reply:
You're right, it's not needed any more. I think it used to be used for the current limiting on the PolarBear or something, I'm not sure though.

Reply:
Oh no never mind, this is actually used in KoalaBear. If you look at the setup functions from electrical, you'll see SPI functions being used....